code/game/machinery/dna_infuser/organ_sets/rat_organs.dm

```diff
@@ -0,0 +1,130 @@
+///bonus of the rat: you can ventcrawl!
+/datum/status_effect/organ_set_bonus/rat
+	organs_needed = 4
+	bonus_activate_text = "Rodent DNA is deeply infused with you! You've learned how to traverse ventilation!"
+
+/datum/status_effect/organ_set_bonus/rat/enable_bonus()
+	. = ..()
+	ADD_TRAIT(owner, TRAIT_VENTCRAWLER_NUDE, REF(src))
+
+/datum/status_effect/organ_set_bonus/rat/disable_bonus()
+	. = ..()
+	REMOVE_TRAIT(owner, TRAIT_VENTCRAWLER_NUDE, REF(src))
+
+
+
+///way better night vision, super sensitive. lotta things work like this, huh?
+/obj/item/organ/internal/eyes/night_vision/rat
+	name = "mutated rat-eyes"
+	desc = "Rat DNA infused into what was once a normal pair of eyes."
+	flash_protect = FLASH_PROTECTION_HYPER_SENSITIVE
+	eye_color_left = "#000000"
+	eye_color_right = "#000000"
+
+/obj/item/organ/internal/eyes/night_vision/rat/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/noticable_organ, "has deep, shifty black pupils, surrounded by a sickening yellow sclera.", BODY_ZONE_HEAD)
```

Comment:
does this mean if i wear a mask it wont show what changed with my eyes?

Reply:
yep